backend/src/request.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+from fastapi import HTTPException
+
+
+def process_include(include: Optional[list[str]], allowed: set[str]):
```

Comment:
Nice idea -- no changes needed here, but we may need to apply this to the routes that I made (unless you hooked this up as middleware).

Reply:
It can't be done as middleware since it requires an argument of which includes are allowed. Once we're done with the demo I'm going to go through and make sure all the routes are consistent.